Input/tests/Input-specs.js

```diff
@@ -37,6 +103,205 @@ describe('Input Specs', () => {
+
+	test('should activate input on enter', () => {
+		const node = document.body.appendChild(document.createElement('div'));
+		const handleChange = jest.fn();
+
+		const subject = mount(
+			<Input onActivate={handleChange} />,
+			{attachTo: node}
+		);
+		const input = subject.find('input');
+
+		input.simulate('keyDown', {which: 13, keyCode: 13, code: 13});
+		input.simulate('keyUp', {which: 13, keyCode: 13, code: 13});
+		node.remove();
+
+		const expected = 1;
+		const actual = handleChange.mock.calls.length;
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should have dir equal to rtl when there is rtl text', () => {
+		const subject = mount(
+			<Input value="שועל החום הזריז קפץ מעל הכלב העצלן.ציפור עפה השעועית עם שקי" />
+		);
+
+		const expected = 'rtl';
+		const actual = subject.find('input').prop('dir');
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should have dir equal to ltr when there is ltr text', () => {
+		const subject = mount(
+			<Input value="content" />
+		);
+
+		const expected = 'ltr';
+		const actual = subject.find('input').prop('dir');
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should have dir equal to rtl when there is rtl text in the placeholder', () => {
+		const subject = mount(
+			<Input placeholder="שועל החום הזריז קפץ מעל הכלב העצלן.ציפור עפה השעועית עם שקי" />
+		);
+
+		const expected = 'rtl';
+		const actual = subject.find('input').prop('dir');
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should have dir equal to ltr when there is ltr text in the placeholder', () => {
+		const subject = mount(
+			<Input placeholder="content" />
+		);
+
+		const expected = 'ltr';
+		const actual = subject.find('input').prop('dir');
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should have dir equal to rtl when there is ltr text in the placeholder, but rtl text in value', () => {
+		const subject = mount(
+			<Input
+				placeholder="content"
+				value="שועל החום הזריז קפץ מעל הכלב העצלן.ציפור עפה השעועית עם שקי"
+			/>
+		);
+
+		const expected = 'rtl';
+		const actual = subject.find('input').prop('dir');
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should have dir equal to ltr when there is rtl text in the placeholder, but ltr text in value', () => {
+		const subject = mount(
+			<Input
+				placeholder="שועל החום הזריז קפץ מעל הכלב העצלן.ציפור עפה השעועית עם שקי"
+				value="content"
+			/>
+		);
+
+		const expected = 'ltr';
+		const actual = subject.find('input').prop('dir');
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should display invalid message if it invalid and invalid message exists', () => {
+		const subject = mount(
+			<Input invalid invalidMessage="invalid message" />
+		);
+
+		expect(subject.find('Tooltip').prop('children')).toBe('invalid message');
+	});
+
+	test('should not display invalid message if it is valid', () => {
+		const subject = mount(
+			<Input invalidMessage="invalid message" />
+		);
+
+		expect(subject.find('Tooltip')).toHaveLength(0);
+	});
+
```

Comment:
This is not a change from this PR, but there are duplicated Tests ('should pause spotlight when input has focus') of Line 321.

When test cases are imported from Sandstone, there are also missing tests. ('should not activate input on enter when disabled')

I feel difficult to review this, so it would be appreciated if you match the order of Test with Sandstone .